test/functional/Java21Only/build.xml

```diff
@@ -35,6 +35,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 	<property name="LIB" value="asm,testng,jcommander" />
 	<import file="${TEST_ROOT}/TKG/scripts/getDependencies.xml" />
 	<property name="TestUtilities" location="../TestUtilities/src"/>
+	<property name="transformerListener" location="${TEST_ROOT}/Utils/src"/>
```

Comment:
Please insert a space before `/>` (like the rest of this file):
```
	<property name="TestUtilities" location="../TestUtilities/src" />
	<property name="transformerListener" location="${TEST_ROOT}/Utils/src" />
```

Reply:
Fixed as suggested above.